ddtrace/ext/tags.go

```diff
@@ -83,6 +83,10 @@ const (
 	// belongs to should be dropped when set to true.
 	ManualDrop = "manual.drop"
 
+	// AppSecKeep is a tag used by AppSec. The trace will be kept
+	// when this tag is set to true.
+	AppSecKeep = "appsec.keep"
```

Comment:
Let's get rid of this. We don't need it as public API because it's not of any use to the user.